pkg/apps/installer.go

```diff
@@ -35,6 +46,7 @@ type Installer struct {
 // source URL.
 type InstallerOptions struct {
 	Type      AppType
+	Operation Operation
```

Comment:
[nitpick] The Installer interface gets a bit weird with these changes.

 A distracted developper can do  `NewInstaller(.... { Operation : Update }).Delete()`  
How about Installer->AppManager & .Update/Install/Delete -> .Run ?